demo/Views/SeekBarView.vala

```diff
@@ -19,11 +19,93 @@
  */
 
 public class SeekBarView : Gtk.Grid {
+
+    private Gtk.Popover preview_popover;
+    private Gtk.Label preview_label;
+
+    public SeekBarView () {
+        Object (halign: Gtk.Align.CENTER,
+                valign: Gtk.Align.CENTER,
+                margin: 24);
```

Comment:
I don't think setting these properties are needed here, since the application will probably set those manually. In particular the `margin` property.